websites/O/Origin/dist/metadata.json

```diff
@@ -13,16 +13,16 @@
     "fi": "Pelaa upeita PC-pelejä ja pidä yhteyttä ystäviisi yhdessä ja samassa paikassa.",
     "fr": "Jouez à de grands jeux PC et connectez-vous avec vos amis sur une seule et même plateforme.",
     "it": "Gioca ai migliori titoli PC e resta in contatto con i tuoi amici, il tutto in un unico spazio.",
-    "jp": "名作PCゲーム。フレンドとのつながり。 その全てはここに。",
+    "ja_JP": "名作PCゲーム。フレンドとのつながり。 その全てはここに。",
     "ko": "명작 PC 게임, 친구들과의 소통. 한 곳에서 모두 즐기세요!",
     "nl": "Speel geweldige PC-games en maak verbinding met je vrienden, allemaal op één plek.",

```

Comment:
pretty sure that has to be ko_KR, so double check everything again mate xD

Reply:
oof - i'll get to it